cypress/integration/dunning/change_text_dunning_doc.js

```diff
@@ -0,0 +1,134 @@
+/// <reference types="Cypress" />
+
+import { SalesInvoice, SalesInvoiceLine } from '../../support/utils/sales_invoice';
+import { getLanguageSpecific, humanReadableNow } from '../../support/utils/utils';
+import { DocumentActionKey, DocumentStatusKey } from '../../support/utils/constants';
+import { BPartner } from '../../support/utils/bpartner';
+import { DunningCandidates } from '../../page_objects/dunning_candidates';
+import { applyFilters, selectNotFrequentFilterWidget, toggleNotFrequentFilters } from '../../support/functions';
+import { DunningType } from '../../support/utils/dunning_type';
+import { DunningDocuments } from '../../page_objects/dunning_documents';
+import { salesInvoices } from '../../page_objects/sales_invoices';
+
+describe('Create Dunning Documents', function() {
+  const date = humanReadableNow();
+
+  const dunningTypeName = `Dunning ${date}`;
+
+  const businessPartnerName = `Customer Dunning ${date}`;
+  const paymentTerm = 'immediately';
+
+  const salesInvoiceTargetDocumentType = 'Sales Invoice';
+  const productName = 'Convenience Salat 250g';
+  const originalQuantity = 200;
+
+  let siDocumentNumber;
+  let dunningTypeRecordId;
+  let siRecordId;
+  let dunningDocId;
+
+  it('Create dunning type', function() {
+    cy.fixture('settings/dunning_type.json').then(dunningType => {
+      Object.assign(new DunningType(), dunningType)
+        .setName(dunningTypeName)
+        .apply();
+    });
+    cy.getCurrentWindowRecordId().then(recordId => {
+      dunningTypeRecordId = recordId;
+    });
+  });
+
+  it('Create bpartner', function() {
+    cy.fixture('sales/simple_customer.json').then(customerJson => {
+      const bpartner = new BPartner({ ...customerJson, name: businessPartnerName })
+        .setCustomer(true)
+        .setDunning(dunningTypeName)
+        .setPaymentTerm(paymentTerm)
+        .setBank(undefined);
+
+      bpartner.apply();
+    });
+  });
+
+  it('Create sales invoice', function() {
+    cy.fixture('sales/sales_invoice.json').then(salesInvoiceJson => {
+      new SalesInvoice(businessPartnerName, salesInvoiceTargetDocumentType)
+        .addLine(new SalesInvoiceLine().setProduct(productName).setQuantity(originalQuantity))
+        .apply();
+    });
+    cy.completeDocument();
+  });
+  it('Sales Invoice is not paid', function() {
+    cy.getCheckboxValue('IsPaid').then(checkBoxValue => {
```

Comment:
There's also a cypress function: `cy.expectCheckboxValue('name', expectedValue)` which can be used here. (just fyi)

Reply:
done.